xml/ha_storage_protection.xml

```diff
@@ -706,9 +729,16 @@ Timeout (msgwait)  : 10
      cluster services are started.</para>
    </step>
    <step>
-    <para>Restart the cluster services on each node:</para>
-    <screen>&prompt.root;<command>crm cluster restart</command></screen>
+    <para>Restart the cluster services on all nodes at once by using the <option>--all</option>
+     option:</para>
+    <screen>&prompt.root;<command>crm cluster restart --all</command></screen>
     <para> This automatically triggers the start of the SBD daemon. </para>
+    <important>
+     <title>Restart cluster services for SBD changes</title>
+     <para>
+       If any SBD metadata changes, you must restart the cluster services again.
```

Comment:
Also good to mention that, in a running cluster, the customer may want to run ` crm maintenance on` to put the cluster in the maintenance mode let the application running while restart the cluster stack. And, `crm maintenance off` after that.

Reply:
This chapter thoroughly covers probably all about "maintenance tasks":

https://susedoc.github.io/doc-sleha/main/html/SLE-HA-administration/cha-ha-maintenance.html

So a reference here probably would be good enough.

Reply:
Thanks Roger and Yan, I'll add a reference to the maintenance chapter.